src/docs/build-deploy-and-maintain-apps/build-an-application.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Build an application
+
+slug: build-an-application
+
+description: Describes how to build an application as best practices of the platform. 
+
+keywords: OpenShift, build, application, hardening, security, SAST, DAST, PaaS, Risk Mitigation, build best practices, app 
+
+page_purpose: Describes the process to build an application in the private cloud as a Service Platform
+
+audience: technical lead, openshift 101 students, openshift 201 students,  developers
+
+author: TBD
+
+editor: Pilar Solares
+
+content_owner: TBD
+
+sort_order: 1
+---
+
+
+# Build an Application 
+Last updated: **DATE**
+
+Purpose of page description goes here 
+
+## On this page
+* [**LINK**](#link) 
+* [**Related pages**](#related-pages)
+
+<!-- ### End of On this page -->
+
+<!-- ### The following topics are listed as suggestions - still to be discussed with subject matter expert  -->
+
+## Create the code for your application
+-  Choosing the programming language and framework
+- Setting up your development environment 
+- Creating the basic structure of your application 
+- Implementing application logic and functionality
+- Testing and debugging your code 
+## CI/CD Pipeline (link to pipeline templates)
```

Comment:
This is the link to the pipleline templates repo: https://github.com/bcgov/pipeline-templates/tree/main/tekton#tekton-pipelines